tests/unit/helpers/questionnaires.test.js

```diff
@@ -260,7 +271,7 @@ describe('getUserQuestionnaires', () => {
     );
   });
 
-  it('should return questionnaire if questionnaire is not answered by this user', async () => {
+  it('should return questionnaire if questionnaire is not answered for this course', async () => {
```

Comment:
j'ai l'impression que ce cas des tests est inclus dans `should return expectations` et dans `should return end of course` qu'en penses tu ?

Reply:
done